cmd/minikube/cmd/start_flags.go

```diff
@@ -548,6 +552,24 @@ func validateVfkitNetwork(n string, options *run.CommandOptions) string {
 	return n
 }
 
+func getRosetta(driverName string) bool {
+	enabled := viper.GetBool(rosetta)
+	if enabled {
+		if !driver.IsVFKit(driverName) {
+			out.Styled(style.Warning, "Rosetta only supported on for the vfkit driver")
```

Comment:
Keep only `on` or `for`, but not both (`on for`) in the warning message.

Reply:
Thanks, fixing